examples/ssh/iperf3.js

```diff
@@ -27,56 +27,53 @@
 
 // Define SSH configuration
 var sshConfig = {
-    timeout: 20000,
+    timeout: 20000
 };
 
-var downloadSpeed, uploadSpeed, downloadSpeedUDP, uploadSpeedUDP;
+var targetServerUrl = D.getParameter('targetServerUrl');
+var targetServerPort = D.getParameter('targetServerPort');
 
-// Define here your target iPerf3 server host and port
-var targetServers = [
-    { url: "lon.speedtest.clouvider.net", port: 5205 },
-];
 
 // Define the commands to be executed via SSH to retrieve speed data and variable configuration
 var execConfig = [
     {
         id: "download_speed",
         label: "Download speed",
-        command: "iperf3 -t 5 -f m -c {url} -p {port} -R",
+        command: "iperf3 -t 5 -f m -c " + targetServerUrl + " -p " + targetServerPort + " -R",
```

Comment:
Why did you do this change?

This is supposed to be a list parameter (same for the below parts)